zc_common/remote_resource/tests.py

```diff
@@ -1,3 +1,4 @@
+import exceptions
```

Comment:
I believe these exceptions are included by default so you don't need to import this module.